azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -1,5 +1,8 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License
+from azure.kusto.data._models import KustoResultTable
+
```

Comment:
I used TYPE_CHECKING whenever an import would cause an error like circular dependency,
It's not needed here, but I can add it for consistency if you think it's right.

Reply:
I like consistency :)